_docs/zh/tools/shell.md

```diff
@@ -3,733 +3,1034 @@ permalink: /docs/tools/shell/
 redirect_from: /overview/shell/
 ---
 
-Pegasus提供了Shell工具，用于查看集群相关信息，创建/删除表，操作数据等。
+Pegasus 提供了 Shell 工具，用于查看集群相关信息，创建/删除表，操作数据等。目前最新版本为 2.5.0。
```

Comment:
It's not needed to mention what the latest version is, it will be outdate if new version released.
You can say, "This document is based on Pegasus 2.5.0" or somthing like this.